docs/source/rst/onboarding/index.rst

```diff
@@ -4,4 +4,264 @@
 Onboarding Guide
 ======================
 
-Stuff about onboarding
+If you are considering contributing, welcome!!
+
+rootski is run like a tech startup, so the onboarding process feels similar
+to starting a new job.
+
+The onboarding process will guide you through...
+
+- 💬 introducing yourself
+- ⚡️ getting access to systems (Slack, ClickUp, GitHub, AWS, etc.)
+- ⚙️ setting up your development environment
+- 🧱 familiarizing yourself with the rootski architecture and our development practices
+- 😎 picking up your first project!
+- 📚 (if you want) creating a learning plan to get the skills you need for your project
+
+The process is completely asynchronous so you can take it as fast or slow as you like 🙂.
+
+Why contribute?
+-----------------
+
+.. dropdown:: ⚠️ Disclaimer, please read!
+   :title: bg-warning
+   :animate: fade-in
+
+   Contributing is 100% "free labor" in the sense that Eric Riddoch owns the rootski project.
+
+   **Please *do* or please *don't* contribute based on what you think is best for your career and
+   personal growth.**
+
+   The value of contributing to rootski is in getting to work on part of a
+   real product, having mentorship, and experimenting with tools you don't use at your day job.
+   Whether or not you're experienced, that "mentorship" looks exactly the way it would at a paid
+   job: we're a bunch of non-experts working together, some of us knowing more about certain technologies
+   than others.
+
+   For Eric, this project is about learning/mentoring. He'd prefer to move slowly
+   so that people can learn rather than push people beyond what they are comfortable with.
+
+   Eric pays $20+/mo (sometimes $50) on this and earns nothing. If that ever changes,
+   he will be clear about that. If anyone were to be paid to work on this, the project
+   simply wouldn't be possible. Eric is happy if the work we're doing here does nothing
+   more than give people career opportunities and help remove confusion about how "real"
+   software is built.
+
+   Many of the contributors make sacrifices offering their time and mentorship working
+   on this project. Please don't criticize our motives. We truly do want you do do
```

Comment:
Got a typo here. Should be this below I'm guessing.

We truly do want you **_to_** do

Reply:
Fixing this myself since this branch is merged already